blocks/property-result-listing/property-result-listing.css

```diff
@@ -1,5 +1,57 @@
 @import url('../property-listing/cards/cards.css');
 @import url('../property-search-bar/search-results-dropdown.css');
+@import url('../property-result-map/map.css');
+
+.property-result-listing.block {
+  display: flex;
+  flex-wrap: wrap;
+}
+
+.property-result-listing.block .search-results-loader {
+  position: relative;
+  overflow: hidden;
+  z-index: 0;
+}
+
+.property-result-listing.block .search-results-loader-image.enter {
+  opacity: 1;
+}
+
+.property-result-listing.block .search-results-loader-image.exit {
+  opacity: 0;
+  pointer-events: none;
+  z-index: -1;
+}
+
+.property-result-listing.block .search-results-loader-image {
+  text-align: center;
+  margin: 0 !important;
+  z-index: 1;
+  height: 100%;
+  width: 100%;
+  display: flex;
+  align-items: flex-start;
+  justify-content: center;
+  background-color: var(--white);
+  transition: all 1s ease-in;
+}
+
+/* .property-result-listing.block .search-results-loader-image img { */
```

Comment:
If this isn't needed, lets get rid of it.

Reply:
fixed